hypha/apply/funds/templates/funds/applicationsubmission_detail.html

```diff
@@ -46,7 +46,7 @@ <h1 class="mb-0 mt-2 font-medium">{{ object.title }}</h1>
                     </a>
 
                     <a class="tab__item" href="#communications" data-tab="tab-2">
-                        {% trans "Communications" %}
+                        {% trans "Communications" %} ({{ object.activities.comments.count }})
```

Comment:
I belive this should be the count of visible comment, no all the comments. I see have a submission with only one comment but the count is 2. 

![Screenshot 2023-10-27 at 9  48 05@2x](https://github.com/HyphaApp/hypha/assets/236356/35d61c38-d12b-4242-8491-365c61e00577)

Reply:
Good catch. Need to filter the count on what comments the user can see. Or we only show it to staff.